src/plugins/view-helpers.ts

```diff
@@ -0,0 +1,30 @@
+import type App from '../app.js';
+import type {InspectOptions} from 'util';
+import type {MojoContext, MojoStash} from '../types.js';
+import {inspect} from 'util';
+
+export default function viewHelpersPlugin (app: App): void {
+  app.decorateContext('inspect', (object: MojoStash, options: InspectOptions) => inspect(object, options));
+
+  app.addHelper('include',
+    (ctx: MojoContext, options: MojoStash, stash: MojoStash) => ctx.renderToString(options, stash));
```

Comment:
`options` should be a real `MojoRenderOptions` in `types.js` with all possible values as optional.